tests/test_fields.py

```diff
@@ -21,12 +23,28 @@
 )
 
 
+if PY3:
+    unicode = str
+
+
+class TestUtilsMixin(object):
+    def assertCountEqual(self, a, b):
+        """Assert a and b are equal in length."""
+        return len(a) == len(b)
+
+
+@contextmanager
+def sub_test(field):
+    """Implement sub_test."""
+    yield
```

Comment:
How does this work? `self.subTest` adds the ability for a test to fail once per iteration in python 3. Does that still work with this?